site/operational-guide.markdown

```diff
@@ -0,0 +1,384 @@
+# Concepts
+
+This section describes some essential concepts with which you will
+need to be familiar before moving on to the example deployment
+scenarios.
+
+> Author's Note: I have deliberately excluded any mention of
+> weave command syntax in this section
+
+## Peer
+
+A peer is a host on which you have installed weave.
```

Comment:
I'd probably say "A peer is a running instance of weave.  Normally you would have one peer on each host".